configdb/hardware/models.py

```diff
@@ -12,21 +12,27 @@ class Meta:
 
 
 class Site(BaseModel):
-    active = models.BooleanField(default=True)
-    code = models.CharField(max_length=3)
-    name = models.CharField(default='', blank=True, max_length=200)
-    lat = models.FloatField(default=0.0)
-    long = models.FloatField(default=0.0)
-    elevation = models.IntegerField(help_text='meters')
+    active = models.BooleanField(default=True, help_text='Whether the site is active and able to accept observations')
+    code = models.CharField(max_length=3, help_text='3-letter site code')
+    name = models.CharField(default='', blank=True, max_length=200, help_text='Site name')
+    lat = models.FloatField(default=0.0, help_text='Site latitude in decimal degrees')
+    long = models.FloatField(default=0.0, help_text='Site longitude in decimal degrees')
+    elevation = models.IntegerField(
+        validators=[MinValueValidator(0), MaxValueValidator(100000)], 
+        help_text='Site elevation in meters'
+    )
     # TODO FIXME:
     # The timezone field does not accurately represent the world. It is supposed
     # to represent the offset from UTC in hours. In many timezones, this changes
     # several times per year (US/Pacific varies between -7 and -8, depending on
     # daylight savings time). It is also incapable of representing many time zones
     # around the world that have non-Integer offsets (example: Australia/Adelaide).
-    timezone = models.IntegerField()
+    timezone = models.IntegerField(
+        help_text='Offset from UTC in hours',
+        validators=[MinValueValidator(-12), MaxValueValidator(12)]
```

Comment:
+12 to -12 seemed like the correct range?

Reply:
yeah that seems appropriate

Reply:
Actually surprisingly it looks like the range is -12 to +14 https://en.wikipedia.org/wiki/List_of_UTC_time_offsets This article has some timezone fun facts and it shows locations that go beyond +12 https://jakubmarian.com/5-weird-things-you-didnt-know-about-time-zones/

Reply:
Unless this timezone field is not meant to represent real timezones but rather be more representative of longitude? Not sure we actually use this field anywhere though